website/source/docs/internals/providers/new-data-source.html.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: "docs"
+page_title: "Creating Data Sources"
+sidebar_current: "docs-internals-provider-guide-new-data-source"
+description: |-
+  How to get started adding a data source to an existing provider.
+---
+
+# Creating Data Sources
+
+Once we have a Provider, we can add some Data Sources to that provider, which
```

Comment:
There's a lot of "we" and "you" here. You are writing technical guides, so this is generally frowned upon.